### PR TITLE
Integrate plumber for running R Codes

### PR DESCRIPTION
**Pull request by danangmassandy**
This is for:

- #666 
- #668
- #712 
- #669 

 from parent ticket #656.

### CODE REVIEW

File: deployment/plumber/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+# [Optional] Set the default user. Omit if you want to keep the default as root.
+USER $USERNAME
+
+FROM prod as dev
+
+# This section taken on 2 July 2015 from
+# https://docs.docker.com/examples/running_ssh_service/
+# Sudo is needed by pycharm when it tries to pip install packages
+RUN apt-get update && apt-get install -y openssh-server sudo
```
Comment (timlinux) [inline deployment/plumber/Dockerfile:75]: Really bad smell from this @danangmassandy  - we dont want to be running stuff with embeded ssh daemons in them. Please approach our implementation with a similar security mindset as we have in our georepo work...

Comment (danangmassandy) [reply deployment/plumber/Dockerfile:75]: I think we can remove the multistage build for this Dockerfile. We use multistage build for the Dockerfile that is used for production, vscode, and pycharm development. It seems we don't need development stage for plumber dockerfile.

**Comment by danangmassandy** [comment]
Configure statistical model:


- 1 - 3: Fill model taxon, name and R code
- 4: Add output type that will be generated from R code
- 5: Download sample data template (hardcoded values)
- 6: Display expected output in JSON based on selected outputs

Preview of national trend in landing page:

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 11 min

File: deployment/docker-compose.override.template.yml [hunk 2]
```diff
@@ -53,6 +55,21 @@ services:
...
+      target: prod
+    volumes:
+      - ../django_project:/home/web/django_project
+      - ./volumes/plumber_data:/home/web/plumber_data
+      - ./volumes/media:/home/web/media
+    links:
+      - db
+      - redis
+    ports:
+      - "${PLUMBER_PORT:-8181}:8181"
```
Comment (dimasciput) [inline deployment/docker-compose.override.template.yml:71]: We don't need to expose the port.

File: django_project/species/views.py [hunk 3]
```diff
@@ -16,3 +16,16 @@ def get(self, request):
             status=200,
             data=TaxonSerializer(taxon, many=True).data
         )
+
+
+class TaxonFrontPageListAPIView(APIView):
```
Comment (dimasciput) [inline django_project/species/views.py:21]: Is this related to this ticket?

Comment (danangmassandy) [reply django_project/species/views.py:21]: Yeah, this is for ticket #669 showing national trends on landing page. Previously, the species list was hardcoded and I changed it to pull from API.

File: deployment/docker-compose.override.template.yml (hunk 2 above)
Comment (danangmassandy) [reply deployment/docker-compose.override.template.yml:71]: alright, I will remove the ports

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 12 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 14 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 13 min
